darc/darc_test.go

```diff
@@ -750,17 +754,25 @@ func TestParseIdentity(t *testing.T) {
 }
 
 // Test any identity
-func testIdentity(t *testing.T, sig Signer) {
-	msg := []byte("something secret")
-	signed, err := sig.Sign(msg)
-	require.NoError(t, err)
+func testIdentity(t *testing.T, id Identity) {
+	msg := []byte(`Hello World`)
+
+	//Signature from code example go-tsm-sdk corresponding to ecdsa public key example
+	signed, _ := hex.DecodeString("304402204f0b20a44efacec7b0514683233a79552026fe80e468078f6fed6cfe3f3e8a0402201eb12db7f6fe0828cafe8b0a032a37ff377b342799cfe77cfbac40c8ec1fa9e8")
```

Comment:
```suggestion
	signed, err := hex.DecodeString("304402204f0b20a44efacec7b0514683233a79552026fe80e468078f6fed6cfe3f3e8a0402201eb12db7f6fe0828cafe8b0a032a37ff377b342799cfe77cfbac40c8ec1fa9e8")
	require.NoError(t, err)
```

ALWAYS do error checking! You will save yourself a lot of pain!